### DISCUSSION TITLE
Agreeing on Default Width and Height for Popups

### DISCUSSION THREAD

**Discussion by faaraxcabdulaahi** [dugsiiyeinc/dhaqsoPay#64] [dhaqsoPay Extension Developers]
@sharafdin 
@dugsiiyeinc/dhaqsopay-extension-developers 

Asxaabta cabir intee le'eg ayaan ka wada dhigana **popUpka** ?

**Comment by sharafdin** [comment]
@hayle102 akhii arinkaas ka discuss gareeya oo isla fadhiisiya Insha'Allah!

**Comment by hayle102** [comment]
Asxaabta waxaan ka cudur daaranaya dib dhaca igu yimid ka raali noqda.

waxan idn soo gudbinaya cabirada saxda ah ee extension-ka

General or container width: 361px
General or container height: 533px

modal width and height:
Width: 300px
height: auto

height wuxu la socon doona xogta uu soo bandhigi doono modal-ka 
sidoo kale waxan sameyay mid example ah aan ku hubinaayo cabirka modal-ka

waakan:
![image](https://github.com/user-attachments/assets/4d517e04-96f2-45b4-a8cd-4c6f59b08aba)

waxaad ku dari kartaan wixi fakarkiina ah oo ah in laga wayneeyo ama laga yareeyo so dhowaada.
